style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  list-style-type: none;
+  text-decoration: none;
+  color: white;
+  margin: 0;
+  padding: 0;
+}
+
+button {
+  background: none;
+  border: none;
+}
+
+body {
+  background-color: #191765;
+}
+
+.pg-hdr {
+  width: 375px;
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  justify-content: space-between;
+  margin-bottom: 47px;
+  padding: 24px;
+}
+
+#main {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  height: auto;
+}
+
+#sct1 {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.titles {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.titles * {
+  width: max-content;
+  height: auto;
+}
+
+.titles h1 {
+  color: #7d73fc;
+  font-size: 48px;
+  font-weight: 700;
+  margin-bottom: 30px;
+}
+
+.titles h2 {
+  font-size: 32px;
+  font-weight: 500;
+  margin-bottom: 70px;
+}
+
+#sct1 .para {
+  text-align: center;
+  font-size: 20px;
+  font-weight: 400;
+  width: 327px;
+  color: #c8c8d4;
+  border-bottom: 2px solid #646369;
+  padding-bottom: 33px;
+  margin-bottom: 37px;
+}
+
+/* Soft Skills */
+.skills {
+  width: 100%;
+  height: auto;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.skills::after {
+  content: " ";
+  width: 88%;
+  height: 2px;
+  background-color: #646369;
+  margin: auto;
+  margin-bottom: 65px;
+}
+
+.competences { margin-left: 43px; }
+
+.cmp-ttl { margin-bottom: 24px; }
+
+.cmp-ttl h3 {
+  font-size: 12px;
+  font-weight: 500;
+  color: #9490ff;
+}
+
+.competence {
+  margin-bottom: 21px;
+  font-size: 15px;
+  font-weight: 400;
+}
+
+/* Socials */
+.join {
+  width: 100%;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 28px;
+}
+
+.Socials {
+  width: 200px;
+  height: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+/* Transition button */
+.btn-work {
+  width: 77px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
```

Comment:
- [ ] For the UX (user experience) purposes, let us fix the following features:

* The button (e.g., "See project") is a link a or button tag, and follow the button styling: Active, Hover and Pressed ([see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9))
* The button (e.g., "Get my resume") is a link a or button tag, and follow the button styling: Active, Hover and Pressed ([see design)](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9)

Reply:
I submitted this by mistake. Sorry!